packages/manager/apps/pci-instances/src/components/input/Input.component.tsx

```diff
@@ -0,0 +1,70 @@
+import { DetailedHTMLProps, InputHTMLAttributes, FC } from 'react';
+import { Input as DatatrInput } from '@datatr-ux/uxlib';
+import { OsdsButton, OsdsIcon } from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_ICON_NAME,
+  ODS_ICON_SIZE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { clsx } from 'clsx';
+
+type InputProps = {
+  handleClear?: () => void;
+  handleValidate?: () => void;
+  isValid?: boolean;
+} & Omit<
+  DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>,
+  'ref'
+>;
+
+export const Input: FC<InputProps> = ({
+  handleClear,
+  handleValidate,
+  isValid = true,
+  className,
+  ...props
+}) => (
+  <div className="flex">
+    <DatatrInput
+      autoFocus
+      className={clsx(
+        'focus-visible:ring-transparent focus-visible:bg-primary-300 text-primary-100',
+        className,
+      )}
+      {...props}
+    />
+    <div className="flex">
+      <OsdsButton
+        data-testid="cancel-change"
+        color={ODS_THEME_COLOR_INTENT.primary}
+        variant={ODS_BUTTON_VARIANT.ghost}
+        size={ODS_BUTTON_SIZE.sm}
+        onClick={handleClear}
```

Comment:
If you have onChange you could send an empty value directly or do you want to handle another way to clear ?